index.html

```diff
@@ -2804,6 +2804,23 @@ <h1>DID Methods</h1>
         <td>
             <a href="https://github.com/docknetwork/dock-did-driver/blob/master/Dock%20DID%20method%20specification.md">Dock DID Method</a>
         </td>
+      </tr>
+       <tr>
+        <td>
+          did:opmn:
+        </td>
+        <td>
+          PROVISIONAL
+        </td>
+        <td>
+            Ethereum, Hyperledger Fabric, Corda
+        </td>
+        <td>
+          <a href="https://dlt.mobi/">dlt.mobi</a>
+        </td>
+        <td>
+            <a href="https://github.com/mobi-dlt/omn/blob/master/OMN-specification.pdf">OMN DID Method</a>
```

Comment:
The specification does not contain the following required sections for a DID Method specification:

* Security Considerations
* Privacy Considerations
* Any detail on how to do Create, Read, Update or Deactivate.

These sections need to be added to the linked to specification before the entry can be added to the registry.

Reply:
Thanks for review! I'm going to ask specification writers to add required sections to the spec and let you know once published.